google/cloud/security/inventory/pipelines/load_projects_iam_policies_pipeline.py

```diff
@@ -36,21 +36,6 @@ class LoadProjectsIamPoliciesPipeline(base_pipeline.BasePipeline):
     RESOURCE_NAME = 'project_iam_policies'
     RAW_RESOURCE_NAME = 'raw_project_iam_policies'
 
-    def __init__(self, cycle_timestamp, configs, crm_client, dao):
-        """Constructor for the data pipeline.
-
-        Args:
-            cycle_timestamp: String of timestamp, formatted as YYYYMMDDTHHMMSSZ.
-            configs: Dictionary of configurations.
-            crm_client: CRM API client.
-            dao: Data access object.
-
-        Returns:
-            None
-        """
-        super(LoadProjectsIamPoliciesPipeline, self).__init__(
```

Comment:
Why is this being removed?  Is this not used for the base pipeline, and later referenced here as self.dao ?
https://github.com/GoogleCloudPlatform/forseti-security/blob/master/google/cloud/security/inventory/pipelines/base_pipeline.py